src/client/QXmppCallManager.cpp

```diff
@@ -117,15 +117,15 @@ QXmppCallPrivate::QXmppCallPrivate(QXmppCall *qq)
 
 QXmppCallPrivate::Stream *QXmppCallPrivate::findStreamByMedia(const QString &media)
 {
-    foreach (Stream *stream, streams)
+    for (auto *stream : streams)
         if (stream->media == media)
             return stream;
     return 0;
 }
 
 QXmppCallPrivate::Stream *QXmppCallPrivate::findStreamByName(const QString &name)
 {
-    foreach (Stream *stream, streams)
+    for (auto *stream : streams)
```

Comment:
ditto (and in a bunch of other places that follow in this file). Let me know if you'd consider this something you'd like to change, and I'll mark the appropriate locations.

Reply:
I think I got most of them, but feel free to check again :)